Add OpenAI OpenAPI Spec, Generate Client Code, Types, and Utilities, and Include Sanitations (#2)
G5andeepD/module-ballerinax-openai-images: docs/spec/sanitations.md

_Authors_: @G5andeepD \
_Created_: 2024/08/05 \
_Updated_: 2024/08/05 \
_Edition_: Swan Lake

# Sanitation for OpenAPI specification

This document records the sanitation done on top of the official OpenAPI specification from OpenAI. The OpenAPI specification is obtained from the [OpenAI OpenAPI Documentation](https://github.com/openai/openai-openapi/blob/master/openapi.yaml). These changes are implemented to enhance the overall usability and readability of the generated client.

> 1. **Removed the `default:null` property of certain schemas**:

Reviewer: ```suggestion
1. **Removed the `default:null` property from the below schemas**:
```
